app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchAllPeopleScreen.kt

```diff
@@ -137,25 +178,38 @@ private fun SearchResult(
                     showMoreOrLessButtonVisible = isSearchActive,
                     onShowAllButtonClicked = searchPeopleScreenState::toggleShowAllPublicResult,
                     onOpenUserProfile = onOpenUserProfile,
+                    expanded = publicResultsExpanded,
+                    onExpansionChanged = onPublicResultsExpansionChanged,
                 )
             }
         }
+//        SideEffect {
+//            if (lazyListState.firstVisibleItemIndex == 0 && lazyListState.firstVisibleItemScrollOffset == 0) {
+//                lazyListState.requestScrollToItem(
+//                    index = lazyListState.firstVisibleItemIndex,
+//                    scrollOffset = lazyListState.firstVisibleItemScrollOffset
+//                )
+//            }
+//        }
+
```

Comment:
👀

Reply:
Right 🤦 I finally extracted that into a separate class but forgot that I commented that instead of simply removing 😅